app/forms/idv/api_image_upload_form.rb

```diff
@@ -290,7 +301,8 @@ def update_analytics(client_response:, vendor_request_time_in_ms:)
           async: false,
           flow_path: params[:flow_path],
           vendor_request_time_in_ms: vendor_request_time_in_ms,
-        ).merge(acuant_sdk_upgrade_ab_test_data).
+        ).except(:classification_info).
```

Comment:
Not sure why I follow - why was `.except(:classification_info)` necessary here?

Reply:
@eileen-nava , it's to exclude logging the classification info in the other event, kind of lazy here to avoid touching many test cases and logging twice.